internal/pkg/cli/svc_delete.go

```diff
@@ -52,14 +60,15 @@ type deleteSvcOpts struct {
 	deleteSvcVars
 
 	// Interfaces to dependencies.
-	store     store
-	sess      sessionProvider
-	spinner   progress
-	prompt    prompter
-	sel       configSelector
-	appCFN    svcRemoverFromApp
-	getSvcCFN func(session *awssession.Session) wlDeleter
-	getECR    func(session *awssession.Session) imageRemover
+	store         store
+	sess          sessionProvider
+	spinner       progress
+	prompt        prompter
+	sel           configSelector
+	appCFN        svcRemoverFromApp
+	getSvcCFN     func(sess *awssession.Session) wlDeleter
+	getECR        func(sess *awssession.Session) imageRemover
+	newSvcDeleter func(sess *awssession.Session, typ string) wkldDeleter
```

Comment:
I wonder if after the `clean` rename, if this distinction will be easier to follow:
```suggestion
	newResourceCleaner func(sess *awssession.Session, typ string) cleaner
```